shared/domains/chelonia/time-sync.js

```diff
@@ -21,7 +21,7 @@ const syncServerTime = async function () {
   // Get our current monotonic time
   const startTime = performance.now()
   // Now, ask the server for the time
-  const time = await fetch(`${this.config.connectionURL}/time`, { signal: this.abortController.signal })
+  const time = await this.config.fetch(`${this.config.connectionURL}/time`, { signal: this.abortController.signal })
```

Comment:
Just comment: I think this is safe because `syncServerTime.call(this)` is used elsewhere... double-check though.

Reply:
What are you referring to as 'safe'? Are you saying whether `this.config` is defined? In that case, it's safe, and the previous code already had `this.config.connectionURL`.

Reply:
Yeah 👍